BuildHelpers/BuildHelpers.psm1

```diff
@@ -39,4 +39,26 @@ Export-ModuleMember -Function Get-Metadata, Update-Metadata, Export-Metadata
 # Set aliases (#10)
 Set-Alias -Name Set-BuildVariable -Value $PSScriptRoot\Scripts\Set-BuildVariable.ps1
 Set-Alias -Name Get-NextPSGalleryVersion -Value Get-NextNugetPackageVersion
-Export-ModuleMember -Alias Set-BuildVariable, Get-NextPSGalleryVersion
+# Backwards compatibilty to command names prior to #72
+Set-Alias -Name Get-BuildVariables -Value Get-BuildVariable
+Set-Alias -Name Get-ModuleAliases -Value Get-ModuleAlias
+Set-Alias -Name Get-ModuleFunctions -Value Get-ModuleFunction
+Set-Alias -Name Set-ModuleAliases -Value Set-ModuleAlias
+Set-Alias -Name Set-ModuleFormats -Value Set-ModuleFormat
+Set-Alias -Name Set-ModuleFunctions -Value Set-ModuleFunction
+Set-Alias -Name Set-ModuleTypes -Value Set-ModuleType

```

Comment:
I would recommend having all these declared in the actual function files themselves, using the `[Alias()]` attribute. Set-Alias consumes more time on import if I recall correctly, and it's much easier in my experience to maintain the aliases for a given command if they're all right there with the command they refer to.

Reply:
I believe it works in V3 and up, but it might even go back to V2.